src/Sulu/Component/CustomUrl/Manager/TitleExistsException.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\CustomUrl\Manager;
+
+use Sulu\Component\Rest\Exception\RestException;
+
+/**
+ * Thrown when a title already exists.
+ */
+class TitleExistsException extends RestException
```

Comment:
Maybe call that one `TitleAlreadyExistsException`, in order to be consistent with the other exceptions of that type?